javascript/chronometer.js

```diff
@@ -1,26 +1,59 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0
+    this.intervalId = ''
```

Comment:
doesn't prevent the code from running or anything but the interval ID returned from the setInterval is always a number, to it makes sense you initialise it as a number instead of a string.